packages/pagination/examples/basic.md

```diff
@@ -93,3 +95,181 @@ initialState = {
   </Row>
 </Grid>;
 ```
+
+### Cursor Pagination
+
+#### Previous & Next
+
+```jsx
+const { Well } = require('@zendeskgarden/react-notifications/src');
+const { getColor } = require('@zendeskgarden/react-theming/src');
+
+const StyledCircle = styled.div`
+  margin: 0 ${props => props.theme.space.base}px;
+  border-radius: 100%;
+  height: ${props => props.theme.space.base * 3}px;
+  width: ${props => props.theme.space.base * 3}px;
+`;
+
+const CircleStroke = styled(StyledCircle)`
+  border: ${props =>
+    `0.6px ${props.theme.borderStyles.solid} ${getColor('grey', 300, props.theme)}`};
```

Comment:
Add `box-sizing: border-box;` to `StyledCircle` and change the border-size to `1px`. Even better, add:

```css
box-sizing: border-box;
border: ${props => props.theme.borders.sm} transparent;
```
to `StyledCircle` and `CircleStroke` would only be:

```css
border-color: ${props => getColor('grey', 300, props.theme)};
```

Reply:
Added. Thanks for the suggestion 👍.